<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DstSrConfig</class>
 <widget class="QWidget" name="DstSrConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="dstGrpBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>351</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="title">
    <string>距离传感器设置</string>
   </property>
   <widget class="QLabel" name="portIdLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>串口号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portIdCbBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="baudRateLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRateCbBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>12</number>
    </property>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>134</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>76800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="dataBitsLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dataBitsCbBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="parityLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>校验位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="parityCbBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>N</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="stopBitsLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="stopBitsCbBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="rwDelayLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>读写超时</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rwDelaySpBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QLabel" name="didLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID设定</string>
    </property>
   </widget>
   <widget class="QLabel" name="rwAdsLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地址设定</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rwAdsSpBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>128</number>
    </property>
   </widget>
   <widget class="QLabel" name="timeLb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>读取间隔</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rwItvalSpBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="dstIdLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
   </widget>
   <widget class="QComboBox" name="nameCbBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="applyBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
